pkg/log/log_messages.go

```diff
@@ -79,4 +79,18 @@ const CAKC063 string = "CAKC063 Unable to find configuration for URL: %s"
 const CAKC064 string = "CAKC064 Unable to find Authenticator for this configuration"
 const CAKC065 string = "CAKC065 File path is invalid %s"
 const CAKC066 string = "CAKC066 Performing authn-jwt"
-const CAKC067 string = "CAKC067 Failed reading jwt token from %s"
+const CAKC067 string = "CAKC067 Failed reading JWT from %s"
+const CAKC068 string = "CAKC068 Loading configuration object..."
+const CAKC069 string = "CAKC069 Choosing configuration object...."
+const CAKC070 string = "CAKC070 Chosen %q configuration"
+const CAKC071 string = "CAKC071 Gathering input settings for configuration..."
+const CAKC072 string = "CAKC072 Successfully gathered input settings"
+const CAKC073 string = "CAKC073 Validating input configuration..."
+const CAKC074 string = "CAKC074 Validated input configuration successfully"
+const CAKC075 string = "CAKC075 Chosen %q Authenticator"
```

Comment:
Since this outputs the authn code name (authn-jwt) and note "JWT", I think we should remove "Authenticator"

Should say either:
"CAKC075 Chosen JWT Authenticator"

or

"CAKC075 Chosen authn-jw

Reply:
It could also Chosen authn-k8s

But i will remove the word authenticator

Reply:
And use the word flow